feature/bgp/policybase/otg_tests/import_export_multi_test/import_export_multi_test.go

```diff
@@ -0,0 +1,535 @@
+// Copyright 2023 Google LLC
```

Comment:
If this test is just moved form one dir to another then can you move this file as well;

Create the new dir then move everything from one dir to another.

`mv dir/* dir_test/`